client/css/style.css

```diff
@@ -2004,8 +1999,20 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 		margin-top: 60px !important;
 	}
 
+	#sidebar button,
+	#sidebar .chan,
+	#sidebar .sign-out,
+	#sidebar .empty,
+	#windows label,
+	#windows .header .topic,
+	#settings .error,
+	#help .help-item,
+	#help .about,
+	#loading,
+	#context-menu,
+	.textcomplete-menu,
 	.messages .msg {
-		font-size: 16px;
+		font-size: 15px;
```

Comment:
Since there are so many overrides, which elements keep the 16px font?

Reply:
None, 16px was set to `.messages .msg`, which is now set to 15px. Everything else is set to 15px on mobile (they are all 14px on desktop).

Reply:
So why aren't you just setting the font-size on body or something?

Reply:
Because at the moment, aaaaaall those things are overriding stuff everywhere. In the long run, I'd like to do that, body or base text be at a given size, and then do specific sizes for other elements (titles, etc., even better if all in `rem`) so that only overriding the base would be enough.
But I won't be able to do this before v2.5.0, and it's a rather extensive change, so I'm getting this out of the way and we can improve later.

Reply:
Font sizes are a real mess right now. I don't intend to fix that with this PR :)